examples/kokkos-tutorials/workload/bfs_bottomup.py

```diff
@@ -0,0 +1,164 @@
+from typing import Tuple
+
+import pykokkos as pk
+
+from parse_args import parse_args
+
+@pk.workload
+class Workload:
+    def __init__(self, N: int, M: int):
+        self.N: int = N
+        self.M: int = M
+
+        self.val: pk.View1D[pk.double] = pk.View([N*M], pk.double)
+        self.vis: pk.View1D[pk.double] = pk.View([N*M], pk.double)
```

Comment:
is vis for visited? if yes, I would suggest expanding the name and using pk.int type